components/admin_console/permission_schemes_settings/permission_group.tsx

```diff
@@ -147,15 +155,15 @@ export default class PermissionGroup extends React.PureComponent {
                 selected={this.props.selected}
                 selectRow={this.props.selectRow}
                 readOnly={this.props.readOnly || comesFromParent}
-                inherited={comesFromParent ? this.props.parentRole : null}
+                inherited={comesFromParent ? this.props.parentRole : undefined}
```

Comment:
Per convention, it's better to assign this value before render as to avoid computations on render.